node-notes-cli/update.js

```diff
@@ -0,0 +1,12 @@
+const data = require('./data.json');
+const fs = require('fs');
+
+function updateNotes() {
+  data.notes[process.argv[3]] = process.argv[4];
+  fs.writeFile("data.json", JSON.stringify(data, null, 2), (err) => {
+    if (err) throw err;
+    console.log('Note has been updated');
+  }
+  );
+}
```

Comment:
"User can update a note" should only work if the user specifies a note that already exists. How would you prevent the user from creating a new note by accident?